src/Test/Utilities/Portable/TestResource.resx

```diff
@@ -209,6 +209,9 @@ namespace My
             }
         }
 
+        public static bool IsAsciiLetter(char c) => c is (>= 'a' and &lt;= 'z') or (>= 'A' and &lt;= 'Z');
```

Comment:
This appears to be only partially XML escaped. Is it only necessary to escape the '<' and not the '>'?

Reply:
Correct.

---
In reply to: [352880294](https://github.com/dotnet/roslyn/pull/39913#discussion_r352880294) [](ancestors = 352880294)